app/views/admin/chat_channels/index.html.erb

```diff
@@ -29,34 +29,46 @@
       <% end %>
     </div>
   </div>
-  <%= paginate @group_chat_channels %>
-  <table class="table">
-    <thead>
+</div>
+<%= paginate @group_chat_channels %>
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col">Channel Name</th>
+      <th scope="col">Users</th>
+      <th scope="col">Add Users</th>
+      <th scope="col">Remove User</th>
+      <% if current_user.any_admin? %>
```

Comment:
I noticed this method is called twice here in this view. I went in a console to see if `has_role?` was doing any local caching and it was not. If we want to improve performance a little here we could add an instance variable for `any_admin?` in the user model like this

```ruby
  def any_admin?
    @any_admin ||= (has_role?(:super_admin) || has_role?(:admin))
  end
```

This would allow us to only hit the database once, instead of twice for this view. It would also improve the performance of a couple of other views I found that do the same. Totally outside the scope of this PR but I wanted to point it out bc I think it's a little peek into how an SRE approaches a PR vs someone looking for correctness. Let me know if you plan to do it, otherwise, I will probably throw up a quick PR for it.

Reply:
@mstruve Thanks for this Molly! I will do so 😄